docs/getting-started/community-membership.sh

```diff
@@ -0,0 +1,119 @@
+:# KubeEdge Community membership
+
+**Note :** This document keeps changing based on the status and feedback of KubeEdge Community.
+
+This document gives a brief overview of the KubeEdge community roles with the requirements and responsibilities associated with them.
+
+| Role | Requirements | Responsibilities | Privileges |
+| -----| ---------------- | ------------ | -------|
+| [Member](#member) | Sponsor from 2 approvers, active in community, contributed to KubeEdge | Welcome and guide new contributors | KubeEdge GitHub organization Member |
+| [Approver](#approver) | Sponsor from 2 maintainers, has good experience and knowledge of domain, actively contributed to code and review  | Review and approve contributions from community members | Write access to relevant repository |
+| [Maintainer](#maintainer) | Sponsor from 2 owners, shown good technical judgement in feature design/development and PR review | Participate in release planning and feature development/maintenance | Write access to relevant repository. Name entry in Maintainers file of the repository |
```

Comment:
So Approvers have the same access permission with Maintaiers? I was thinking take Maintainers as the top-level Approvers and Approvers only have the write access to spefiic packages.

Reply:
Yes, approvers have only "write access" to specific packages which should be enforced by bot, since github only have repo level read/write access management.